mlrun/run.py

```diff
@@ -744,11 +744,10 @@ def update_common(fn, spec):
         raise ValueError("Databricks tasks only support embed_code=True")
 
     if kind == RuntimeKinds.application:
-        if handler:
-            raise MLRunInvalidArgumentError(
-                "Handler is not supported for application runtime"
-            )
-        filename, handler = ApplicationRuntime.get_filename_and_handler()
+        raise MLRunInvalidArgumentError(
+            "Code to function is not supported for application runtime. "
+            "Code can be specified via project/function source."
```

Comment:
```suggestion
            "Embedding a code file is not supported for application runtime. "
            "Code files should be specified via project/function source."
```

I'm not sure this message is really explaining the situation very well (especially now that users are not expected to call `code_to_function` anymore so this sounds strange). TBH my suggestion doesn't sound very good to me either, but it's a bit more explicit.